data_fetcher.py

```diff
@@ -0,0 +1,110 @@
+import os
+import shutil
+import json
+import pandas as pd
+import yfinance as yf
+from datetime import datetime
+
+class DataFetcher:
+    """
+    This module downloads data from yahoo finance and saves it locally in a downloads directory
+    """
+    def __init__(self, directory_path: str, import_start_date: datetime=datetime(2008,1,1)) -> None:
+        self.module_version = 1
+        self.directory_path = directory_path
```

Comment:
is it the data directory? I think the nameing can be better

Reply:
done